### DISCUSSION TITLE
How to add an image block or text block in a database.

### DISCUSSION THREAD

**Discussion by ybee2003** [ramnes/notion-sdk-py#103] [Q&A] [answered]
I wonder how to add image blocks or text to the database page.

In notion-py, I added it using
`addimage = row.children.add_new(ImageBlock,width=600)`

In notion-sdk-py, I know how to add rows, 
`notion.pages.create(parent={"database_id": database_id}, properties=new_page)`
but I don't know how to add images or text in row page.

Please understand that I used a translator because I am not good at English.

**Comment by AxDSan** [comment, marked as answer]
```python
<...>
image_block_props= {
  "type": "image",
  "image": {
    "type": "external",
    "external": {
        "url": "https://website.domain/images/image.png"
    }
  }
}
<...>
notion.pages.create(parent={"database_id": database_id}, properties=new_page, children=[image_block_props])
```

That's it! you were pretty close! ⭐

 I welcome you to read more upon:
* [Block Keys](https://developers.notion.com/reference/block#block-object-keys)
* [Block Objects](https://developers.notion.com/reference/block#block-type-object)
* [Image Blocks](https://developers.notion.com/reference/block#image-blocks)
